documentation/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "scripts": {
+    "start": "sweet-potato-cooker start",
+    "build": "sweet-potato-cooker build",
+    "clean": "sweet-potato-cooker clean",
+    "compress": "sweet-potato-cooker compress"
```

Comment:
Need to remove this. We don't compress with the cooker anymore. That should also be updated on the peeler templates.

Reply:
Ah yeah that's straight off the minimal template. I'll update that in another PR